commands/mta_command.go

```diff
@@ -13,7 +13,7 @@ import (
 	"github.com/cloudfoundry/cli/cf/formatters"
 	"github.com/cloudfoundry/cli/cf/terminal"
 	"github.com/cloudfoundry/cli/plugin"
-	"github.com/cloudfoundry/cli/plugin/models"
+	plugin_models "github.com/cloudfoundry/cli/plugin/models"
```

Comment:
Why putting name on the import?

Reply:
It is autogenerated from visual studio code and its extensions :/